tests/unittests/quantizationTest.cpp

```diff
@@ -138,6 +138,37 @@ TEST(Quantization, quantizeGraph) {
   EE.run({}, {});
 }
 
+/// Quantize ReLU node and make sure that quantized version
+/// has quantization parameters mapping to non-negative floating
+/// point range.
+TEST(Quantization, quantizeReLU) {
+  ExecutionEngine EE;
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+  auto *input = mod.createVariable(ElemKind::FloatTy, {1, 3}, "input");
```

Comment:
nit: perhaps make this `VisibilityKind::Public`, because in theory we could add a new optimization later that folds a `max(splat, private Variable)` into the Variable (assuming the quantize node is also folded into the Variable), and so this test would break.